### PR TITLE
feat: Add GetSecret and RunTimeSecret metrics

### PR DESCRIPTION
**Pull request by geraldzm**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

## Testing Instructions

[go-mod-bootstrap](https://github.com/edgexfoundry/go-mod-bootstrap/pull/530)

### CODE REVIEW

File: cmd/core-common-config-bootstrapper/res/configuration.yaml [hunk 1]
```diff
@@ -19,6 +19,8 @@ all-services:
         SecuritySecretsStored: false
         SecurityConsulTokensRequested: false
         SecurityConsulTokenDuration: false
+        securityRuntimeSecretTokenDuration: true
+        SecurityGetSecretDuration: true
```
Comment (bnevis-i) [inline cmd/core-common-config-bootstrapper/res/configuration.yaml:23]: Set to `false` once go.mod is upgraded to latest dev version.

File: cmd/core-common-config-bootstrapper/res/configuration.yaml [hunk 2]
```diff
@@ -19,6 +19,8 @@ all-services:
         SecuritySecretsStored: false
         SecurityConsulTokensRequested: false
         SecurityConsulTokenDuration: false
+        securityRuntimeSecretTokenDuration: true
```
Comment (lenny-intel) [inline cmd/core-common-config-bootstrapper/res/configuration.yaml:22]:
```suggestion
        SecurityRuntimeSecretTokenDuration: true
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by bnevis-i — CHANGES_REQUESTED** [review]
Need to bump the go-mod-bootstrap version.  Also should have just force pushed on top instead of creating the additional commit.

**Review by bnevis-i — APPROVED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**